Leanplum/Vendor/UniqueIdentifier/UIDevice+IdentifierAddition.m

```diff
@@ -0,0 +1,117 @@
+//
+//  UIDevice(Identifier).m
+//  UIDeviceAddition
+//
+//  Created by Georg Kitz on 20.08.11.
+//  Copyright 2011 Aurora Apps. All rights reserved.
+//  Copyright (C) 2012, Georg Kitz, @gekitz
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy of
+//  this software and associated documentation files (the "Software"), to deal in
+//  the Software without restriction, including without limitation the rights to
+//  use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies
+//  of the Software, and to permit persons to whom the Software is furnished to do
+//  so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in all
+//  copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+//  SOFTWARE.
+
+#import "UIDevice+IdentifierAddition.h"
+#import "NSString+MD5Addition.h"
+
+#include <sys/socket.h> // Per msqr
+#include <sys/sysctl.h>
+#include <net/if.h>
+#include <net/if_dl.h>
+
+@interface UIDevice(Private)
+
+- (NSString *) macaddress;
+
+@end
+
+@implementation UIDevice (IdentifierAddition)
+
+////////////////////////////////////////////////////////////////////////////////
+#pragma mark -
+#pragma mark Private Methods
+
+// Return the local MAC addy
+// Courtesy of FreeBSD hackers email list
+// Accidentally munged during previous update. Fixed thanks to erica sadun & mlamb.
+- (NSString *) macaddress{
```

Comment:
do we need this at all for post-iOS 7+?

Reply:
yea looks like we do not need this. Great catch. This helping clean up .